guides/Git/command-git.md

```diff
@@ -0,0 +1,7 @@
+# Description of Git :
+
+Git is a technologie created by Linus Torvalds. Git is inspired by SVN or other solutions of version management, but SVN is hard to handle. So Mr Torvald wanted to make a more simple and easy to handle solution for developpers. 
+
+Has we said Git is a version management, so it helps you to manage all your developpement time of your project to save step by step your code. Git allow you to get back to all the version you saved. 
+
+
```

Comment:
There are these few typos and some commands you need to write on. For example, what does `git pull` do, `git push origin master` do, etc.